apps/codelab/src/app/multiselect/multiselect.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  Component,
+  Provider,
+  QueryList,
+  Type,
+  ViewChild,
+  ViewChildren
+} from '@angular/core';
+import { MultiselectModule } from './multiselect.module';
+import { MultiselectModel } from './multiselect-model';
+import { MultiselectListDirective } from './multiselect-list.directive';
+import { MultiselectItemDirective } from './multiselect-item.directive';
+
+@Component({
+  template: `
+    <div [msModel]="selectionModel" multiselectList>
```

Comment:
can the api be `multiselectList="selectionModel"` instead? 
Or maybe even expose selection model from the directive if needed?